.github/workflows/test.yml

```diff
@@ -0,0 +1,60 @@
+name: Test
+
+on: [push, pull_request]
+
+env:
+  FORCE_COLOR: 1
```

Comment:
does anything use this?  this isn't a standard environment variable I'm familiar with

Reply:
At least pytest does, I normally just include it as a default. But it's not being passed via tox and the test output is monochrome, so I'll just remove it.